src/raster/r.vif/r.vif.py

```diff
@@ -382,7 +421,7 @@ def main(options, flags):
         finally:
             text_file.close()
             gs.message("\n")
-            gs.message("Statistics are written to " + OPF + "\n")
+            gs.message("Statistics are written to " + output_file + "\n")
```

Comment:
_verbose_, not _message_ in my view. Also, for core module, this would have to be translatable message formatted with `.format()`. `\n` is not needed. Don't format strings with `+`. Try Pylint, it will tell you to use f-string in this case (will not if you use the translate `_` function like for core).

Reply:
Changed to use format. What is the disadvantage of _message_?